src/testing/ui-test-playwright/playeruserflow.spec.js

```diff
@@ -10,43 +10,68 @@
 
 const { test, expect } = require('@playwright/test');
 
-// If this test file is being used for actual UI testing, set these values both to 1sec
-const minTaskWaitSeconds = process.env.TEST_MIN_TASK_WAIT_SECONDS || 3;
-const maxTaskWaitSeconds = process.env.TEST_MAX_TASK_WAIT_SECONDS || 6;
+const baseUrl = process.env.TEST_BASEURL; // Needs to include protocol, e.g. https://
 
+// If this test file is being used to simulate more realistic user behaviour, change these values
+const minTaskWaitSeconds = process.env.TEST_MIN_TASK_WAIT_SECONDS || 1;
+const maxTaskWaitSeconds = process.env.TEST_MAX_TASK_WAIT_SECONDS || 1;
 
-test('shoppinguserflow', async ({ page }) => {
+// If this test file is being used to simulate more realistic user behaviour, change these values
+const minNumberOfItems = process.env.TEST_MIN_NUMBER_OF_ITEMS || 1;
+const maxNumberOfItems = process.env.TEST_MAX_NUMBER_OF_ITEMS || 1;
 
-    // Header to indicate that posted comments and rating are just for testing and can be deleted again by the app
-    page.setExtraHTTPHeaders({'X-TEST-DATA': 'true'});
+const runAllOptionalSteps = process.env.TEST_RUN_ALL_OPTIONAL_STEPS || false;
 
-    const baseUrl = process.env.TEST_BASEURL; // Needs to include protocol, e.g. https://
+const screenshotPath = process.env.SCREENSHOT_PATH || '';
+const captureScreenshots = screenshotPath != '' ? true : false;
+
+test('shoppinguserflow', async ({ page }) => {
 
-    // If this test file is being used for actual UI testing, set these variables both to the same value
-    const minNumberOfItems = process.env.TEST_MIN_NUMBER_OF_ITEMS || 2;
-    const maxNumberOfItems = process.env.TEST_MAX_NUMBER_OF_ITEMS || 5;
+    // Header to indicate that posted comments and rating are just for testing and can be deleted again by the app
+    page.setExtraHTTPHeaders({ 'X-TEST-DATA': 'true' });
 
     // Go to main page
     await page.goto(baseUrl);
 
+    if (captureScreenshots) {
+        await page.screenshot({ path: `${process.env.SCREENSHOT_PATH}/root.png` });
+    }
+
     // Go to the catalog page
     await page.goto(`${baseUrl}/#/catalog`);
     await expect(page).toHaveURL(`${baseUrl}/#/catalog`);
     await page.waitForTimeout(getRandomWaitTimeMs());
 
+    if (captureScreenshots) {
+        await page.screenshot({ path: `${process.env.SCREENSHOT_PATH}/catalog.png` });
+    }
+
+    // Count the number of items we found so we can randomly select one
+    var catalogItems = await page.$$('div.catalog-item');
+
+    expect(catalogItems.length > 0).toBeTruthy();
```

Comment:
https://vincenttunru.com/toBeTruthy-vs-toBe-true/

I wonder if `expect(catalogItems.length > 0).toBe(true)` is better, given how it works internally :)